training/train.py

```diff
@@ -51,6 +51,28 @@ def load_data(s3_client):
     return raw_data_dict_
 
 
+def check_style(data_dict):
+    """Check the format of manifest file."""
+    if data_dict.get("ecosystem") == str("maven"):
+        package_dict = data_dict.get("package_dict")
+        if package_dict:
+            try:
+                if not all([k in package_dict for k in ("user_input_stack", "bigquery_data")]):
+                    logger.info("Keys are missing.")
+                    return False
+                else:
+                    logger.info("Your manifest is in proper format.")
+                    return True
+            except Exception as e:
+                logger.error('%r' % e)
+        else:
+            logger.error("ERROR - manifest.json is not in proper format.")
+            return False
+    else:
+        logger.error("ERROR - You are not training for maven.")
```

Comment:
you can have logger.info here instead of logger.error.
logger.info("Skipping because the ecosystem is not maven.")

Reply:
Ok Got it!